internal/pkg/agent/cmd/container.go

```diff
@@ -92,11 +92,13 @@ The following actions are possible and grouped based on the actions.
   FLEET_SERVER_ELASTICSEARCH_CA_TRUSTED_FINGERPRINT - The sha-256 fingerprint value of the certificate authority to trust
   FLEET_SERVER_ELASTICSEARCH_INSECURE - disables cert validation for communication with Elasticsearch
   FLEET_SERVER_SERVICE_TOKEN - service token to use for communication with elasticsearch
+  FLEET_SERVER_SERVICE_TOKEN_PATH - path to service token file to use for communication with elasticsearch
   FLEET_SERVER_POLICY_ID - policy ID for Fleet Server to use for itself ("Default Fleet Server policy" used when undefined)
   FLEET_SERVER_HOST - binding host for Fleet Server HTTP (overrides the policy). By default this is 0.0.0.0.
   FLEET_SERVER_PORT - binding port for Fleet Server HTTP (overrides the policy)
   FLEET_SERVER_CERT - path to certificate to use for HTTPS endpoint
   FLEET_SERVER_CERT_KEY - path to private key for certificate to use for HTTPS endpoint
+  FLEET_SERVER_CERT_KEY_PASSPHRASE_PATH - path to private key passphrase file for certificate to use for HTTPS endpoint
```

Comment:
@blakerouse do you think we should change this as well to be consistent with the flags and `FLEET_SERVER_CERT_KEY` or keep as is to make it clear that the var should not be the passphrase value?

Reply:
Yes I think it should be consistent with flags. The description of the ENV will be enough to describe that it should be a path.